### PR TITLE
feat: Add Symfony HttpClient integration tests

### PR DESCRIPTION
**Pull request by pvgnd**
### Context
php-vcr does not seems to work with Symfony HttpClient (#329). Here is the error:  
`RuntimeException: Unexpected error, could not find curl_getinfo in response or errors`

### What has been done

- Integration tests from guzzle have been duplicated for symfony/http-client in order to fix the issue

### How to test

- 

### Todo

- [ ] Find what is the issue and fix it

### CODE REVIEW

**Comment by aaa2000** [comment]
Maybe, you could apply the following PRs #285 #286 #294  to fix the issue

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ notes.md
 tests/fixtures/cassette*
 /.php_cs.cache
 /.php_cs
+composer.lock
```
Comment (JeroenVanOort) [inline .gitignore:9]: I don't remember why composer.lock is tracked, but I would suggest that it's removal is done in a separate PR.

**Review by JeroenVanOort — CHANGES_REQUESTED** [review]
Could you please rebase your branch against the current master? I believe some things got mixed up during the years.

**Comment by JeroenVanOort** [comment]
Would you be so kind to rebase again? And some tests are still failing.

**Comment by pvgnd** [comment]
Yes I'm trying to figure out what is the issue but can't make it work for now

**Comment by pvgnd** [comment]
abandoned

**Pull request closed without merging**